operations/bills.md

```diff
@@ -168,28 +207,42 @@ A bill is either a `Receipt` which means that it has been fully paid, or `Invoic
 | `TrackReceivable` | boolean | required | Tracking of payments is enabled for bill, only applicable for `Invoice`. |
 | `DisplayCid` | boolean | required | Display CID number on bill, only applicable for `Invoice`. |
 
-#### Bill assignee data
+#### Bill owner data
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `Discriminator` | string [Bill assignee data discriminator](#bill-assignee-data-discriminator) | required | Determines type of value. |
-| `Value` | object | required | Structure of object depends on [Bill assignee data discriminator](#bill-assignee-data-discriminator). |
+| `Discriminator` | string [Bill owner data discriminator](#bill-owner-data-discriminator) | required | Determines type of value. |
+| `Value` | object | required | Structure of object depends on [Bill owner data discriminator](#bill-owner-data-discriminator). Can be either of type [Bill customer data](#bill-customer-data) or [Bill company data](#bill-company-data)|
 
-#### Bill assignee data discriminator
+#### Bill owner data discriminator
 
-* `BillCustomerData` - Assignee data specific to a customer.
-* `BillCompanyData` - Assignee data specific to a company.
+* `BillCustomerData` - Owner data specific to a customer.
+* `BillCompanyData` - Owner data specific to a company.
 
 #### Bill customer data
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `ItalianFiscalCode` | string  | optional | Italian fiscal code. |
+| `Id` | string  | optional | ID of the [Customer](customers.md#customer) to whom the bill was assigned. |
+| `Address` | [Bill address](#bill-address)  | optional | Address of the customer. |
+| `LegalIdentifiers` | [LegalIdentifiers](#legal-identifiers) | optional | Legal Identifiers of the customer. |
+| `BillingCode` | string  | optional | A unique code for Mews to list on invoices it sends to the customer. |
+| `LastName` | string  | optional | Last name of the customer. |
+| `FirstName` | string  | optional | First name of the customer. |
+| `SecondLastName` | string  | optional | Second last name of the customer. |
+| `TitlePrefix` | [Title](customers.md#Title)  | optional | Title prefix of the customer. |
 
 #### Bill company data
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
+| `Id` | string  | optional | ID of the [Company](companies.md#company). |
+| `Address` | [Bill address](#bill-address)  | optional | Address of the company. |
+| `LegalIdentifiers` | [LegalIdentifiers](#legal-identifiers) | optional | Legal Identifiers of the company. |
+| `BillingCode` | string  | optional | A unique code for Mews to list on invoices it sends to the company. |
+| `Name` | string  | optional | Name of the company. |
+| `FiscalIdentifier` | string  | optional | Fiscal identifier of the company. |
+| `AdditionalTaxIdentifier` | string  | optional | Additional tax identifier of the company. |
```

Comment:
Can you double check `Contract` here please? I don't think all of them are optional.
There is a discriminator which tells you if there is a customer or company but this doesn't relate to the Contract here. This contract column determinates only if the entity is provided so I think of example `Id` is always required.

Same goes for the customer please.

I hope my poor explanation is good 😅